docs/manual/migration-guide.md

```diff
@@ -0,0 +1,79 @@
+- Proceed similarly for video tracks.
+
+### Unity integration
+
+Unlike the C# library, the Unity integration takes a step back and build some convenience features on top of the transceiver API of the C# library. For the user, this avoids having to deal manually with pairing transceivers and tracks, and relying instead on a much simpler declarative model.
+
+Users are encouraged to follow the updated [Unity tutorial](helloworld-unity.md) to understand how to setup a peer connection with this new API.
+
+#### Peer connection
+
+The [`PeerConnection`](xref:Microsoft.MixedReality.WebRTC.Unity.PeerConnection) component now holds a collection of _media lines_, which can be described as a sort of "transceiver intent". These describe the final result the user intend to produce after an SDP session is fully negotiated in terms of transceivers and their attached tracks. The component internally manages adding transceivers when needed to match the user's media line description.
+
+#### Signaling
+
+- The `PeerConnection.Signaler` property has been removed; the [`Signaler`](xref:Microsoft.MixedReality.WebRTC.Unity.Signaler) component is now only a helper to creating custom signaling solution, but is not required anymore.
+- As a result, the [`Signaler`](xref:Microsoft.MixedReality.WebRTC.Unity.Signaler) component now has a [`Signaler.PeerConnection`](xref:Microsoft.MixedReality.WebRTC.Unity.Signaler.PeerConnection) property which must be set up by the user. This is true in particular for the [`NodeDssSignaler`](xref:Microsoft.MixedReality.WebRTC.Unity.NodeDssSignaler) component which derives from the [`Signaler`](xref:Microsoft.MixedReality.WebRTC.Unity.Signaler) abstract base component.
+
+#### Video
+
+- The `LocalVideoSource` component, which was previously using a webcam to capture video frames, as been renamed into the more explicit [`WebcamSource`](xref:Microsoft.MixedReality.WebRTC.Unity.WebcamSource) component.
+- The [`WebcamSource`](xref:Microsoft.MixedReality.WebRTC.Unity.WebcamSource) component derives from the abstract [`VideoSender`](xref:Microsoft.MixedReality.WebRTC.Unity.VideoSender) component, which is also the base class for the callback-based sources like the [`SceneVideoSender`](xref:Microsoft.MixedReality.WebRTC.Unity.SceneVideoSender) component which captures its video frame from the rendering of any Unity Camera component.
+- The `RemoteVideoSource` component has been renamed into the [`VideoReceiver`](xref:Microsoft.MixedReality.WebRTC.Unity.VideoReceiver) component.
+- The `VideoSource` component is now an interface :  [`IVideoSource`](xref:Microsoft.MixedReality.WebRTC.Unity.IVideoSource).
```

Comment:
nit: should probably say what to do if one has an instance of `VideoSource` in their scene (not sure if this is even possible)

Reply:
`VideoSource` was abstract anyway.